### ISSUE TITLE
[jtreg]jtreg-jdk11 TestGCBasherWithShenandoah.java fail because of compile error

### ISSUE THREAD

**Issue by sendaoYan** [adoptium/aqa-tests#3935]
**Describe the bug**

```log
/home/yansendao/git/dragonwell11/test/hotspot/jtreg/gc/stress/gcbasher/TestGCBasherWithShenandoah.java:148: error: cannot find symbol
        TestGCBasher.main(args);
        ^
  symbol:   variable TestGCBasher
  location: class TestGCBasherWithShenandoah
1 error

TEST RESULT: Failed. Compilation failed: Compilation failed
```

**To Reproduce**
jtreg test/hotspot/jtreg/gc/stress/gcbasher/TestGCBasherWithShenandoah.java

**Additional context**
fix patch:
```diff
diff --git a/test/hotspot/jtreg/gc/stress/gcbasher/TestGCBasherWithShenandoah.java b/test/hotspot/jtreg/gc/stress/gcbasher/TestGCBasherWithShenandoah.java
index d7e2aaa0212..5f6f830c9ed 100644
--- a/test/hotspot/jtreg/gc/stress/gcbasher/TestGCBasherWithShenandoah.java
+++ b/test/hotspot/jtreg/gc/stress/gcbasher/TestGCBasherWithShenandoah.java
@@ -32,6 +32,7 @@ import java.io.IOException;
  * @requires vm.gc.Shenandoah & !vm.graal.enabled
  * @requires vm.flavor == "server" & !vm.emulatedClient & !vm.graal.enabled
  * @summary Stress the Shenandoah GC by trying to make old objects more likely to be garbage than young objects.
+ * @compile TestGCBasher.java ClassInfo.java Decompiler.java Bytecode.java  ByteCursor.java  ConstantPoolEntry.java  Dependency.java  MethodInfo.java
  *
  * @run main/othervm/timeout=200 -Xlog:gc*=info -Xmx1g -XX:+UnlockDiagnosticVMOptions -XX:+UnlockExperimentalVMOptions
  *      -XX:+UseShenandoahGC -XX:ShenandoahGCMode=passive
@@ -52,6 +53,7 @@ import java.io.IOException;
  * @requires vm.gc.Shenandoah & !vm.graal.enabled
  * @requires vm.flavor == "server" & !vm.emulatedClient & !vm.graal.enabled
  * @summary Stress the Shenandoah GC by trying to make old objects more likely to be garbage than young objects.
+ * @compile TestGCBasher.java ClassInfo.java Decompiler.java Bytecode.java  ByteCursor.java  ConstantPoolEntry.java  Dependency.java  MethodInfo.java
  *
  * @run main/othervm/timeout=200 -Xlog:gc*=info -Xmx1g -XX:+UnlockDiagnosticVMOptions -XX:+UnlockExperimentalVMOptions
  *      -XX:+UseShenandoahGC -XX:ShenandoahGCHeuristics=aggressive
@@ -76,6 +78,7 @@ import java.io.IOException;
  * @requires vm.gc.Shenandoah & !vm.graal.enabled
  * @requires vm.flavor == "server" & !vm.emulatedClient & !vm.graal.enabled
  * @summary Stress the Shenandoah GC by trying to make old objects more likely to be garbage than young objects.
+ * @compile TestGCBasher.java ClassInfo.java Decompiler.java Bytecode.java  ByteCursor.java  ConstantPoolEntry.java  Dependency.java  MethodInfo.java
  *
  * @run main/othervm/timeout=200 -Xlog:gc*=info -Xmx1g -XX:+UnlockDiagnosticVMOptions -XX:+UnlockExperimentalVMOptions
  *      -XX:+UseShenandoahGC -XX:ShenandoahGCHeuristics=adaptive
@@ -95,6 +98,7 @@ import java.io.IOException;
  * @requires vm.gc.Shenandoah & !vm.graal.enabled
  * @requires vm.flavor == "server" & !vm.emulatedClient & !vm.graal.enabled
  * @summary Stress the Shenandoah GC by trying to make old objects more likely to be garbage than young objects.
+ * @compile TestGCBasher.java ClassInfo.java Decompiler.java Bytecode.java  ByteCursor.java  ConstantPoolEntry.java  Dependency.java  MethodInfo.java
  *
  * @run main/othervm/timeout=200 -Xlog:gc*=info -Xmx1g -XX:+UnlockDiagnosticVMOptions -XX:+UnlockExperimentalVMOptions
  *      -XX:+UseShenandoahGC -XX:ShenandoahGCHeuristics=compact
@@ -109,6 +113,7 @@ import java.io.IOException;
  * @requires vm.gc.Shenandoah & !vm.graal.enabled
  * @requires vm.flavor == "server" & !vm.emulatedClient & !vm.graal.enabled
  * @summary Stress the Shenandoah GC by trying to make old objects more likely to be garbage than young objects.
+ * @compile TestGCBasher.java ClassInfo.java Decompiler.java Bytecode.java  ByteCursor.java  ConstantPoolEntry.java  Dependency.java  MethodInfo.java
  *
  * @run main/othervm/timeout=200 -Xlog:gc*=info -Xmx1g -XX:+UnlockDiagnosticVMOptions -XX:+UnlockExperimentalVMOptions
  *      -XX:+UseShenandoahGC -XX:ShenandoahGCMode=iu -XX:ShenandoahGCHeuristics=aggressive
@@ -133,6 +138,7 @@ import java.io.IOException;
  * @requires vm.gc.Shenandoah & !vm.graal.enabled
  * @requires vm.flavor == "server" & !vm.emulatedClient & !vm.graal.enabled
  * @summary Stress the Shenandoah GC by trying to make old objects more likely to be garbage than young objects.
+ * @compile TestGCBasher.java ClassInfo.java Decompiler.java Bytecode.java  ByteCursor.java  ConstantPoolEntry.java  Dependency.java  MethodInfo.java
  *
  * @run main/othervm/timeout=200 -Xlog:gc*=info -Xmx1g -XX:+UnlockDiagnosticVMOptions -XX:+UnlockExperimentalVMOptions
  *      -XX:+UseShenandoahGC -XX:ShenandoahGCMode=iu
```

[issue3935.patch.txt](https://github.com/adoptium/aqa-tests/files/9424668/issue3935.patch.txt)

**Comment by smlambert** [comment]
@andrew-m-leonard - can this get reported/fixed upstream?

**Comment by sendaoYan** [comment]
> @andrew-m-leonard - can this get reported/fixed upstream?

This problem has been fix by patch 9d0b5e59f41dd6ed6b6eca57996346b038863c68

![image](https://user-images.githubusercontent.com/24123821/187193098-fed34d91-1ca7-4362-b016-4dfb7c33320f.png)
